features/step_definitions/networking.rb

```diff
@@ -1180,3 +1180,35 @@
       raise "Failed to clear mtu field from CNO" unless @result[:success]
   }
 end
+
+Given /^I save egress data file directory to the#{OPT_SYM} clipboard$/ do | cb_name |
+  ensure_admin_tagged
+  cb_name = "cb_egress_directory" unless cb_name
+  network_operator = BushSlicer::NetworkOperator.new(name: "cluster", env: env)
+  network_type = network_operator.network_type(user: admin)
+  case network_type
+  when "OVNKubernetes"
+    cb[cb_name]="ovn-egressfirewall"
+  when "OpenShiftSDN"
+    cb[cb_name]="egressnetworkpolicy"
+  else
+    raise "unknown network_type"
+  end
+  logger.info "The egressfirewall fie directory path is stored to the #{cb_name} clipboard."
+end
+
+Given /^I save egress type to the#{OPT_SYM} clipboard$/ do | cb_name |
+  ensure_admin_tagged
+  cb_name = "cb_egress_type" unless cb_name
+  network_operator = BushSlicer::NetworkOperator.new(name: "cluster", env: env)
+  network_type = network_operator.network_type(user: admin)
+  case network_type
+  when "OVNKubernetes"
+    cb[cb_name] = "egressfirewall"
+  when "OpenShiftSDN"
+    cb[cb_name] = "egressnetworkpolicy"
+  else
+    raise "unknown network_type"
+  end
+  logger.info "The egressnetwork policy fie directory path is stored to the #{cb_name} clipboard."
```

Comment:
It looks like most of the steps are the same, perhaps we should merge those two step definitions ?

Reply:
Not all the cases need both steps, some case may just need one step here, so I think it is ok to keep it as it is.

Reply:
Then you need to update this line. It is a simple copy without modification to adapt.

Reply:
aha, sorry for these typo, updated it!